client/projects/cms/src/app/elements/new-prepopulate/new-prepopulate.component.ts

```diff
@@ -26,16 +26,16 @@ export class NewPrepopulateComponent implements OnInit {
   icon: string;
 
   @Input()
-  dataset: string;
+  data: string;
 
   @Input()
   collection: string;
-  data: any;
+  private _data: any;
```

Comment:
We don't really use this notation for private properties I don't think we should have it here exclusively.